google/cloud/aiplatform/datasets/datasources.py

```diff
@@ -0,0 +1,141 @@
+import abc
+from typing import Optional, Dict, Sequence, Union
+from google.cloud.aiplatform_v1beta1 import (
```

Comment:
These should be module level imports and they should be imported directly from the module they are implemented in.